### DISCUSSION TITLE
after completion all the process are given in session still i did'nt get github student developer pack

### DISCUSSION THREAD

**Discussion by gauravjejurkar** [github/india#315] [General]
i had completed all the process are share in the session but still not recive any student developer pack can anyone tell me how claim it

**Comment by AbhishekMondal1** [comment]
Did you get the claim notification in self-learning section?

**Reply by ABHISHEKNBR10** [reply]
I have submitted the request. How long will it take??

**Reply by gauravjejurkar** [reply]
I think 3 working days

**Comment by gauravjejurkar** [comment]
Yes

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
